### DISCUSSION TITLE
GameCube Blueretro Bluetooth disconnects

### DISCUSSION THREAD

**Discussion by George304** [darthcloud/BlueRetro#818] [General]
Hello,

I've been trying to update the GameCube blueretro. Every time it connects and start the update process it disconnects and never finishes. Please help, I can't update the firmware at all. It keeps disconnecting.

**Comment by SaltyAsFrig** [comment]
I have the same issue. I can connect to other pages (advanced, presets, system) most of the time, but OTA consistently disconnects. Like this:
![Screenshot 2023-11-07 215017](https://github.com/darthcloud/BlueRetro/assets/49793830/8c56fe5c-7d82-4d65-8748-c60a4c640ea4)


From the system page I see this version:
Connected to: BlueRetro_GC_5AE2 (c0:49:ef:91:5a:e2) [v1.8-rc4 hw2 gamecube]

I've already tried factory reset and it makes no difference either.

**Reply by BPHusker** [reply]
This is exactly my problem.  I've tried with a PC and a laptop and I'm getting the same error message.

**Comment by darthcloud** [comment]
For those of you that got the issue that only the OTA page doesn't work (the other connect fine) I made a fix for that. Please try again.

**Reply by SaltyAsFrig** [reply]
Thank you so much for fixing whatever the issue was. I was able to update to 1.9 HW2 flawlessly on the first try tonight. 

Cheers!
